UITests/Receipts/PrepaidCardListReceiptTests.swift

```diff
@@ -0,0 +1,248 @@
+                             filename: "PrepaidCardReceiptsForOneMonth",
+                             method: HTTPMethod.get)
+
+        XCTAssertTrue(prepaidCardReceiptMenu.exists)
+        prepaidCardReceiptMenu.tap()
+        waitForNonExistence(spinner)
+        if #available(iOS 12, *) {
+            verifyCellLabels(with: "Balance Adjustment\nJun 23, 2019", moneyTitle: "-7.00\nUSD", by: 2)
+        } else {
+            verifyCellLabels(with: "Balance Adjustment Jun 23, 2019", moneyTitle: "-7.00 USD", by: 2)
+        }
+    }
+
+    func testPrepaidCardReceiptsList_userHasNoTransactions() {
+        mockServer.setUpEmptyResponse(url: "/rest/v3/users/usr-token/prepaid-cards/trm-token/receipts")
+        XCTAssertTrue(prepaidCardReceiptMenu.exists)
+        prepaidCardReceiptMenu.tap()
+        waitForNonExistence(spinner)
+
+        XCTAssertTrue(app.staticTexts["Seems like, you don’t have any Transactions, yet."].exists)
+        XCTAssertEqual(app.tables.cells.count, 0)
+    }
+
+    // MARK: Detail View
+    func testPrepaidReceiptDetail_verifyCredit() {
+        mockServer.setupStub(url: "/rest/v3/users/usr-token/prepaid-cards/trm-token/receipts",
+                             filename: "PrepaidCardReceiptsForOneMonth",
+                             method: HTTPMethod.get)
+        XCTAssertTrue(prepaidCardReceiptMenu.exists)
+        prepaidCardReceiptMenu.tap()
+        waitForNonExistence(spinner)
+        transactionDetails.openReceipt(row: 0)
+        waitForExistence(transactionDetails.detailHeaderTitle)
+        // Transaction
+        XCTAssertEqual(transactionDetails.transactionSection.label, "Transaction")
+        if #available(iOS 12, *) {
+            XCTAssertEqual(transactionDetails.paymentLabel.label, "Funds Deposit\nJun 20, 2019")
+            XCTAssertEqual(transactionDetails.paymentAmountLabel.label, "10.00\nUSD")
+        } else {
+            XCTAssertEqual(transactionDetails.paymentLabel.label, "Funds Deposit Jun 20, 2019")
+            XCTAssertEqual(transactionDetails.paymentAmountLabel.label, "10.00 USD")
+        }
+
+        // Details
+        XCTAssertEqual(transactionDetails.detailSection.label, "Details")
+        XCTAssertEqual(transactionDetails.receiptIdLabel.label, "Receipt ID:")
+        XCTAssertEqual(transactionDetails.dateLabel.label, "Date:")
+        XCTAssertEqual(transactionDetails.receiptIdValue.label, "FISVL_5269000")
+        XCTAssertEqual(transactionDetails.dateValue.label, "Thu, Jun 20, 2019, 9:23 PM")
+
+        // Fee
+        XCTAssertEqual(transactionDetails.feeSection.label, "Fee Specification")
+        XCTAssertEqual(transactionDetails.amountLabel.label, "Amount:")
+        XCTAssertEqual(transactionDetails.transactionLabel.label, "Transaction:")
+        XCTAssertEqual(transactionDetails.amountValue.label, "10.00 USD")
+        XCTAssertEqual(transactionDetails.transactionValue.label, "10.00 USD")
+    }
+
+    func testPrepadReceiptDetail_verifyDebit() {
```

Comment:
typo in testPrepadReceiptDetail_verifyDebit -> testPrepaidReceiptDetail_verifyDebit